src/diffusers/models/transformers/transformer_sd3.py

```diff
@@ -245,6 +245,7 @@ def forward(
         encoder_hidden_states: torch.FloatTensor = None,
         pooled_projections: torch.FloatTensor = None,
         timestep: torch.LongTensor = None,
+        block_controlnet_hidden_states: List = None,
```

Comment:
Perhaps we rename to `transformer_block_additional_residuals` here to keep it consistent with the UNet? cc: @yiyixuxu

Reply:
Oh nvm, it isn't applied at the block level. Please ignore.